### PR TITLE
Move creation of ZIS plugin dataset to configure time

### PR DESCRIPTION
**Pull request by timgerstel**
Signed-off-by: Timothy Gerstel <tgerstel@rocketsoftware.com>

Please check if your PR fulfills the following requirements. This is simply a reminder of what we are going to look for before merging your PR. If you don't know all of this information when you create this PR, don't worry. You can edit this template as you're working on it.

- [ ] Necessary documentation (if appropriate) have been added / updated
- [X] DCO signoffs have been added to all commits, including this PR

#### PR type
What type of changes does your PR introduce to Zowe? _Put an `x` in the box that applies to this PR. If you're unsure about any of them, don't hesitate to ask._ 
- [ ] Bugfix 
- [X] Feature 
- [ ] Other... Please describe:

#### Relevant issues

Fixes https://github.com/zowe/zlux/issues/684

#### Changes proposed in this PR

- Move ZIS dataset creation logic into separate script, invoke during configure time

#### Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

#### Does this PR do something the person installing Zowe should know about?

---
* Affected function: _general area of interest_ *

---
* Description: Create dataset for ZIS plugins if such flags are specified during instance configuration *

---
* Part: _name of customizable file involved_  *

---
_multi-line description_

#### Is there a related doc issue or Pull Request? 

Doc issue/PR number:

### CODE REVIEW

**Review by lchudinov — COMMENTED** [review]
@timgerstel for some reason `${ROOT_DIR}/scripts/zowe-create-ZIS-ds.sh` doesn't exist, so the pax did not install successfully: 
` .: get_zis_params 30: create_new_instance 3: ./zowe-configure-instance.sh 248: /ZOWE/staging/zowe/scripts/zowe-create-ZIS-ds.sh: not found`

Actually, the file was `zowe-1.24.0/scripts/zowe-create-ZIS-ds.sh`.

Can you please fix it.

File: scripts/zowe-create-ZIS-ds.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+echo "Creating dataset for ZIS plugins ... " >> $LOG_FILE
+
+# 1. {datasetprefix}.SZWEPLUG
+
+# TODO replace by allocate-dataset.sh call to resuse VOLSER support
+tsocmd "allocate new da('${ZOWE_DSN_PREFIX}.SZWEPLUG') " \
+    "dsntype(library) dsorg(po) recfm(u) lrecl(0) bLKSIZE(32760)" \
+    "unit(sysallda) $sizePLUG" >> $LOG_FILE 2>&1
+rc=$?
+if test $rc -eq 0
```
Comment (lchudinov) [inline scripts/zowe-create-ZIS-ds.sh:26]: Should we check the case when the dataset already exists, for example with  `LISTDS '${ZOWE_DSN_PREFIX}.SZWEPLUG'` TSO command?

File: scripts/zowe-create-ZIS-ds.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+sizePLUG='space(30,15) tracks'
+
+script=zowe-create-ZIS-ds.sh
+echo "<$script>" >> $LOG_FILE
+echo "Creating dataset for ZIS plugins ... " >> $LOG_FILE
+
+# 1. {datasetprefix}.SZWEPLUG
+
+# TODO replace by allocate-dataset.sh call to resuse VOLSER support
+tsocmd "allocate new da('${ZOWE_DSN_PREFIX}.SZWEPLUG') " \
```
Comment (Martin-Zeithaml) [inline scripts/zowe-create-ZIS-ds.sh:22]: There is a documented behavior for ```tsocmd``` which causes "freezing" of the script when executed via ZOWE CLI, see [#1012](https://github.com/zowe/zowe-cli/issues/1012). I recommend to add the end of the standard input.

**Review by 1000TurquoisePogs — APPROVED** [review]
This now looks good to me after reviewing and testing.

**Pull request merged**